lib/index.js

```diff
@@ -37,47 +37,50 @@ module.exports = {
 		var msgs = {}, msgid = 1, handlers = {}, processMsg, smtpServer, that;
 		smtpServer = smtp.createServer(smtpOptions);
 		smtpServer.on("startData",function(envelope){
-		  envelope.body = "";
+			envelope.body = "";
```

Comment:
Ironically, I have been switching all my JS to spaces over tabs. But I do not have any desire to be religious about it and aggressively change this!